src/panels/lovelace/common/handle-action.ts

```diff
@@ -81,7 +83,7 @@ export const handleClick = (
       }
       const [domain, service] = actionConfig.service.split(".", 2);
       hass.callService(domain, service, actionConfig.service_data);
+      forwardHaptic("success");
```

Comment:
Would have been great if we could have had this fix as part of 101.

Reply:
bram added it

Reply:
https://github.com/home-assistant/home-assistant-polymer/pull/4152

Reply:
🎉